CMakeModules/build_cl2hpp.cmake

```diff
@@ -9,6 +9,7 @@ ExternalProject_Add(
     PREFIX "${prefix}"
     INSTALL_DIR "${prefix}/package"
     UPDATE_COMMAND ""
+    BUILD_BYPRODUCTS "${prefix}/package/CL/cl2.hpp"
```

Comment:
Its BUILD_BYPRODUCTS for this command. I guess it is a newer CMAKE option. I'll take a look.